pom.xml

```diff
@@ -704,5 +704,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
do we really need these? I recall this causing issues previously...

Reply:
https://github.com/castorini/anserini/pull/2597

Reply:
Did this modification on a different branch. Forgot to merge. Sorry and it's now resolved.